src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -57,6 +60,9 @@ class ProjectConfigurationModal extends PureComponent<Props, State> {
     ev.preventDefault();
 
     const { saveProjectSettings, project } = this.props;
+    if (!project) {
+      return;
+    }
```

Comment:
This block should be unnecessary, right? It shouldn't be possible for the user to call `saveSettings` if the modal isn't visible.

Reply:
indeed, but flow isn't happy if it's not there

Reply:
Ah, you're correct.